.pre-commit-config.yaml

```diff
@@ -1,10 +1,10 @@
 repos:
```

Comment:
Updated pre-commit to match the checks we run in CI, as mentioned in https://github.com/django-components/django-components/issues/1395#issuecomment-3350413821.